### PR TITLE
[Designer] Add aria attributes to dropdown controls

### PR DESCRIPTION
**Pull request by paulcam206**
## Related Issue
Fixes VSO #24095594
Fixes VSO #24113643
Fixes VSO #24081494

## Description
A few fixes:
* The "Preview Mode" toolbar button now gets the `aria-pressed` property set appropriately set
* The toolbar dropdown menus are now marked up correctly for accessibility
* Keyboard access to peer comboboxes now works

## How Verified
* local build, narrator, dev tools, keyboard

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/4171)

### CODE REVIEW

File: source/nodejs/adaptivecards-controls/src/inputwithpopup.ts [hunk 1]
```diff
@@ -265,6 +275,13 @@ export abstract class InputWithPopup<TPopupControl extends PopupControl, TValue>
         }
     }
 
+    get labelId(): string {
+        if (this._labelElement) {
+            return this._labelElement.id;
+        }
+        return undefined;
```
Comment (dclaux) [inline source/nodejs/adaptivecards-controls/src/inputwithpopup.ts:282]: Empty line before after } and before return.

File: source/nodejs/adaptivecards-designer/src/toolbar.ts [hunk 2]
```diff
@@ -75,6 +76,7 @@ export class ToolbarButton extends ToolbarElement {
         else {
             this.renderedElement.classList.remove("acd-toolbar-button-toggled");
         }
+        this.renderedElement.setAttribute("aria-pressed", this.isToggled.toString());
```
Comment (dclaux) [inline source/nodejs/adaptivecards-designer/src/toolbar.ts:79]: Empty line after }

File: source/nodejs/adaptivecards-controls/src/dropdown.ts [hunk 3]
```diff
@@ -236,4 +268,12 @@ export class DropDown extends InputWithPopup<DropDownPopupControl, DropDownItem>
             this.selectedItem = this.items.get(index);
         }
     }
+
+    get parentLabelId() : string {
+        return this._parentLabelId;
+    }
+
+    set parentLabelId(newParentLabelId: string) {
```
Comment (dclaux) [inline source/nodejs/adaptivecards-controls/src/dropdown.ts:276]: Nit: I tend to consistently use `value` as the name of parameter in a property setter.

**Comment by shalinijoshi19** [comment]
@paulcam206 I've been labeling the accessibility bug fixes that need to go out as patches to the release/1.2 with AdaptiveCards v1.2.10 label. Please remove this label if this doesn't need to be patched and ported over to release/1.2. Thanks!

**Pull request merged**